### PR TITLE
LPS-133181 - Filter the list of contents by author

### PR DESCRIPTION
**Pull request by Luizcarlosqueiroz**
**JIRA Ticket:**
LPS-133181
LPS-114985

**Motivation:**
Test Automation for "Filter the list of contents by author"

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 1]
```diff
@@ -1564,4 +1564,126 @@ definition {
...
+
+		task ("Check if all contents are being displayed") {
+			ContentDashboard.searchValue(webContentTitle = "WC Title 1");
+
+			AssertTextPresent(
+				locator1 = "ContentDashboard#FIRST_ELEMENT_LISTED",
+				value1 = "WC Title 1");
+
+			ContentDashboard.searchValue(webContentTitle = "WC Title 2");
+
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:1622]: When you search a title, does it keep the filter applied?

Comment (Luizcarlosqueiroz) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:1622]: Yes. It insert the text and 'press' enter, filtering. And in the second searchValue, it overwrites the last one.

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:test:tango

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
@Luizcarlosqueiroz Hey luiz, this PR is failing the SF. Please check the tests again.

**Comment by Luizcarlosqueiroz** [comment]
> @Luizcarlosqueiroz Hey luiz, this PR is failing the SF. Please check the tests again.

@CarlosBrichete I did SF before, but I noticed that there was a test not in alphabetical order. If this wasn't the problem, it might from the others commits?

**Comment by CarlosBrichete** [comment]
ci:test:sf

**Comment by CarlosBrichete** [comment]
@Luizcarlosqueiroz SF is still failing, so maybe you will have to rebase it or something... Try to close this PR and send a new one.

**Comment by CarlosBrichete** [comment]
ci:close

**Comment by Luizcarlosqueiroz** [comment]
New PR:
https://github.com/liferay-tango/liferay-portal/pull/1269

**Pull request closed without merging**